### PR TITLE
OSSM-3750: Incorrect default load balancing policy

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.9+

Issue: https://issues.redhat.com/browse/OSSM-3750

Link to docs preview: https://58930--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-traffic-manage.html#ossm-routing-destination-rules_traffic-management

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rh-tokeefe** [comment]
@luksa and @pbajjuri20 please review the change for this defect. Thanks!

File: modules/ossm-routing-destination-rules.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 Destination rules are applied after virtual service routing rules are evaluated, so they apply to the traffic's real destination. Virtual services route traffic to a destination. Destination rules configure what happens to traffic at that destination.
 
-By default, {SMProductName} uses a round-robin load balancing policy, where each service instance in the pool gets a request in turn. {SMProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
+By default, {SMProductName} uses a least requests load balancing policy, where each service instance in the pool gets a request in turn. {SMProductName} also supports the following models, which you can specify in destination rules for requests to a particular service or service subset.
```
Comment (luksa) [inline modules/ossm-routing-destination-rules.adoc:12]: I'm looking at the resource schema and these are the allowed values: `LEAST_CONN`, `RANDOM`, `PASSTHROUGH`, `ROUND_ROBIN`, and `LEAST_REQUEST`. So, I guess this should be "least request" or maybe "least-request"?

**Review by luksa — APPROVED** [review]
LGTM, but I'm not completely sure about "least requests" vs "least-request".

**Review by pbajjuri20 — APPROVED** [review]

**Comment by gwynnemonahan** [comment]
LGTM

**Comment by rolfedh** [comment]
Petr mentioned that 4.9 is EOL as of today.
https://redhat-internal.slack.com/archives/G01Q6LPP04S/p1681849120285319?thread_ts=1681829936.301389&cid=G01Q6LPP04S
Please update your PR description to specify 4.10 and later. Thanks!

**Pull request merged**